likecoin/includes/class-likecoin-matters-api.php

```diff
@@ -231,12 +231,14 @@ private function post_multipart_query( $query, $variables, $file ) {
 	 *
 	 * @param string| $title Draft title.
 	 * @param string| $html_content Draft HTML content.
+	 * @param array|  $tags Array of tag strings.
 	 */
-	public function new_draft( $title, $html_content ) {
+	public function new_draft( $title, $html_content, $tags ) {
 		$payload  = 'mutation {
       putDraft(input: {
         title: ' . wp_json_encode( $title ) . ',
-        content: ' . wp_json_encode( $html_content ) . ',
+				content: ' . wp_json_encode( $html_content ) . ',
+				tags: ' . wp_json_encode( $tags ) . ',
```

Comment:
Extra indentation?

Reply:
should be tab vs space, force pushed to space